src/gpt-auto-generator/gpt-auto-generator.c

```diff
@@ -759,6 +768,13 @@ static int add_mounts(void) {
         }
 
         return enumerate_partitions(devno);
+
+euclean:
+        return log_error_errno(
+                        r,
+                        "File system behind %s system is reported by btrfs to be backed by pseudo-device /dev/root, which is not a valid user-space accessible device node. "
```

Comment:
"system" seems misplaced here, we'll have "File system behind root file system system".
user-space → userspace for consistency with the rest of the codebase.